### ISSUE TITLE
Making the site more responsive

### ISSUE THREAD

**Issue by mraza007** [PacBSD/pacweb#12]
Hi Can I work on this site and make this more responsive and mobile friendly

**Comment by markzz** [comment]
I say go for it if you are inclined to.

**Comment by mraza007** [comment]
Sure I'll try btw can this project support dell xps13

**Comment by markzz** [comment]
You can certainly try it, just make note that you will need to use a legacy boot instead of EFI.

**Comment by mraza007** [comment]
I see
I'll try otherwise I am going with arch
I like the userland feature of the bsd

**Comment by markzz** [comment]
I strongly suggest you go with Arch unless you're playing around with this. This project has been stagnant for a while now and sees little updates.

**Comment by mraza007** [comment]
You are right that's why I was a little hesitant and the docs were limited too

**Comment by vendion** [comment]
@mraza007 At this point I wouldn't recommend running PacBSD on any real hardware or anything like that. Running it in an isolated VM should be okay though. Not only is the project out of date but our repos where heavily lacking so you may have found it difficult to work with as a daily driver, unless you are willing to help build and package things that you find missing ;)

As for updating the web site have at it, just please keep is something simple to maintain. The current site is a static site built with [Hugo](https://gohugo.io/) using [hugo-theme-arch](https://github.com/syui/hugo-theme-arch) as our base theme.
